src/styles/blocks/footer.css

```
@@ -23,10 +23,19 @@
     color: inherit;
 }
 
-.footer__link[href] {
+.footer__link[href],
+.footer__link[href]:visited {
+    color: var(--color-grey-lightest);
+}
+
+.footer__link[href]:hover {
     color: white;
 }
 
+.footer__link[href]:focus {
```

```suggestion
.footer__link[href]:hover {
```
Состояния фокуса пока в дизайне не описаны, будем делать их отдельно